contrib/runners/inquirer/runner.yaml

```diff
@@ -0,0 +1,37 @@
+- description: A runner for Inquiries
+  enabled: true
+  name: inquirer
+  runner_module: inquirer
+  runner_parameters:
+    schema:
+      default:
+        title: response_data
+        type: object
+        properties:
+          continue:
+           type: boolean
+           description: Would you like to continue the workflow?
+        required: ["continue"]
+      required: true
+      description: A JSON schema that will be used to validate the response data
+      type: object
+    tag:
+      default: ""
```

Comment:
Maybe we should default to None instead?

Reply:
There's no advantage and you can find other inconsistencies around. But I agree w/ @Kami.  For the most part, we default value to None if no value should be assigned. To me, an empty string is still a value.  Sometimes we default an empty string and empty dict because processing None and processing empty list/dict causes a lot of unnecessary logic. Whereas processing a None on a variable of string type is relatively straight forward.